rules/firestore.test.json

```diff
@@ -173,6 +173,7 @@ service cloud.firestore {
                 (playerOneTimeUnchanged || safeTimeIncreasePlayerOne || playerOneTimeDecreased);
             return isValidPartUpdateDuringGame(part) &&
                 safeTimeManagement &&
+                preservesField('winner') && preservesField('loser') &&
```

Comment:
non move updates cannot change winner and loser ?

Reply:
si ils peuvent, c'est pour ça que ces lignes ont été enlevées depuis